Source/Controllers/MainViewControllers/TableRelations/SPTableRelations.h

```diff
@@ -51,7 +51,8 @@
 	IBOutlet NSTextField         *constraintName;
 	IBOutlet NSBox               *addRelationTableBox;
 	IBOutlet NSPopUpButton       *columnPopUpButton;
-	IBOutlet NSPopUpButton       *refTablePopUpButton;
+    IBOutlet NSPopUpButton       *refDatabasePopUpButton;
+    IBOutlet NSPopUpButton       *refTablePopUpButton;
 	IBOutlet NSPopUpButton       *refColumnPopUpButton;
```

Comment:
Inconsistent spacing, looks like

Reply:
Well, on my side, the code looks like ok. I think the problem is my Xcode is configured for using 4 spaces instead of tab character.

Reply:
![XCode_2](https://user-images.githubusercontent.com/7204168/105096314-257e3b00-5aa7-11eb-9ce5-4755206b5736.jpg)
![XCode](https://user-images.githubusercontent.com/7204168/105096319-2747fe80-5aa7-11eb-8aed-ca9bd098bb51.jpg)

Reply:
The problem might be the other code, and your might be correct. The other code previously was differently styled. Just re-indent whole file after the PR is green and approved.